includes/polling/os/junos.inc.php

```diff
@@ -1,5 +1,52 @@
 <?php
 
+use LibreNMS\RRD\RrdDefinition;
+
+$oid_list = 'jnxJsSPUMonitoringCurrentFlowSession.0 jnxJsSPUMonitoringCPUUsage.0 jnxJsSPUMonitoringMemoryUsage.0';
```

Comment:
Can we not get this data into the actual mempool and processors module?

Reply:
@laf  not sure I understand your request, can you explain?

Reply:
So we already have discovery and poller modules that deal with memory and processors so adding them here is counter to that + you can't alert on this data. So, if it's possible to get data from snmp that can be used in those modules it should be done there.